deploy/src/Microsoft.Azure.IIoT.Deployment/Resources/scripts/jumpbox.sh

```diff
@@ -323,7 +323,7 @@ if [ "$is_private_repo" = true ] ; then
     echo $setsvc
 
     # Install aiiot/azure-industrial-iot Helm chart
-    helm install --atomic azure-industrial-iot aiiot/azure-industrial-iot --namespace azure-industrial-iot --version $HELM_CHART_VERSION --timeout 30m0s \
```

Comment:
@marcschier, I would advice to keep the `--atomic` flag here. It makes sure that if Helm chart deployment did not succeed in `30` minutes then a failure is reported. Without it we will have scenarios when providing invalid Helm configuration details (such as invalid image, image tag or version) will not be reported as error and deployment tool will continue without notifying the caller that something went wrong.

Reply:
What I see is that any error that occurs is rolled back and thus difficult to diagnose on the cluster, meaning we would have a hard time telling what went wrong in the field.  But no problem to keep status quo.